tron/kubernetes.py

```diff
@@ -53,6 +54,12 @@ def __init__(self, action_run_id: str, task_config: KubernetesTaskConfig, serial
 
         self.log = self.get_event_logger()
 
+        config_str = str(self.get_config())
+        # AWS_SECRET_ACCESS_KEYs are base64-encoded so it uses alphanumerics plus +, /, and =
+        config_str = re.sub("'AWS_SECRET_ACCESS_KEY': '[a-zA-Z0-9+/=]+'", "AWS_SECRET_ACCESS_KEY_REDACTED", config_str)
+        config_str = re.sub("'AWS_ACCESS_KEY_ID': '[a-zA-Z0-9]+'", "AWS_ACCESS_KEY_ID_REDACTED", config_str)
+        self.log.info(f"Kubernetes task {self.get_kubernetes_id()} created with config {config_str}")
```

Comment:
Is this an accidental commit? Feels like a debugging line to check AWS creds, and not something that should be permanent.

Reply:
I should probably add this in a separate PR - I was just comparing my new "meta" output to what is shown for Mesos and realized that we weren't showing users how their task was launched (this is more-or-less what we do for Mesos tasks)